src/Extraload/Transformer/MapFieldsTransformer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Extraload\Transformer;
+
+class MapFieldsTransformer implements TransformerInterface
+{
+    private $fields;
+
+    public function __construct(array $fields)
+    {
+        $this->fields = $fields;
+    }
+
+    public function transform($data)
```

Comment:
I like this transformer, but we can make it better by using https://symfony.com/doc/current/components/property_access.html and make it work with multi-dimensional arrays and objects. What do you think?

Also, it would be great to extract transformet in a separate PR.